edb/server/config/__init__.py

```diff
@@ -33,19 +34,19 @@
     'spec_to_json', 'to_json', 'from_json',
     'value_from_json',
     'OpLevel', 'OpCode', 'Operation',
-    'ConfigType', 'Port',
+    'ConfigType',
     'load_spec_from_schema',
     'generate_config_query',
 )
 
 
-_settings = None
+_settings = Spec()
```

Comment:
Note: I changed the default because there were many places where the server is using `get_settings()` indiscriminately without checking if the settings are None or not.

In a real scenario this won't happen so initially I put an assert in `get_settings()` to check they are in fact not None. It made a bunch of tests fail which could be worked around (clumsily but functionally). What broke the camel's back for me though was that the compiler depends on the settings to figure out if we are in test mode and if to do constant folding. Working around *that* would be too much trouble so I just changed the default here, all tests pass.